ui/src/features/BottomPanel/components/DataTable/index.tsx

```diff
@@ -1,40 +1,91 @@
-import { useT } from "@flow/providers";
+import { ColumnDef } from "@tanstack/react-table";
+
+import { Checkbox, DataTable as Table } from "@flow/components";
+import { points } from "@flow/mock_data/pointData";
+
+export type Payment = {
+  id: string;
+  amount: number;
+  status: "pending" | "processing" | "success" | "failed";
+  email: string;
+};
+
+type Fire = {
+  ACQ_DATE: string;
+  ACQ_TIME: string;
+  BRIGHT_TI4: number;
+  BRIGHT_TI5: number;
+  CONFIDENCE: string;
+  DAYNIGHT: string;
+  FRP: number;
+  LATITUDE: number;
+  LONGITUDE: number;
+  SATELLITE: string;
+  SCAN: number;
+  TRACK: number;
+  VERSION: string;
+};
+
+const columns: ColumnDef<Fire>[] = [
+  {
+    accessorKey: "ACQ_DATE",
+    header: "ACQ_DATE",
+  },
+  {
+    accessorKey: "ACQ_TIME",
+    header: "ACQ_TIME",
+  },
+  {
+    accessorKey: "BRIGHT_TI4",
+    header: "BRIGHT_TI4",
+  },
+  {
+    accessorKey: "BRIGHT_TI5",
+    header: "BRIGHT_TI5",
+  },
+  {
```

Comment:
This is all boiler so can you at least add a TODO comment stating as much? 
What this DataTable component should end up doing is either receiving processed columns + data (columns based on the data), or receiving data, and based on the data generating the columns.
If you can write that quickly please do, but just a comment is fine now since the main thing is the UI at the moment 🙏🏻

Reply:
Added the comment for clarity.